app/models/user.rb

```diff
@@ -0,0 +1,17 @@
+class User < ApplicationRecord
+  devise :database_authenticatable, :registerable,
+         :recoverable, :rememberable, :validatable
+
+  validates :name, presence: true, length: { maximum: 20 },
+                   format: { with: /\A[a-zA-Z]+\z/, message: '英文字のみが使えます。スペースも禁止です。' }
```

Comment:
細かいけど、禁止っていわれると圧が強いなとおもった
```suggestion
                   format: { with: /\A[a-zA-Z]+\z/, message: '英文字のみが使えます。スペースは使用できません。' }
```